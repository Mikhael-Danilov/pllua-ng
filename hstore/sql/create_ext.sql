--

create extension hstore_pllua cascade;

--end
