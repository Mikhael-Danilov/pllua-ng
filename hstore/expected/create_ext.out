--
create extension hstore_pllua cascade;
NOTICE:  installing required extension "hstore"
NOTICE:  installing required extension "pllua"
--end
